### DISCUSSION TITLE
Where is the upstream of  the rhel-system-roles.sshd Ansible role?

### DISCUSSION THREAD

**Discussion by beachrunner-au** [linux-system-roles/linux-system-roles.github.io#104] [Q&A] [answered]
I would have expected to find a linux-roles.sshd repo under this GitHub org but it doesn't exist. Does an upstream of rhel-system-roles.sshd exist or is it being obtained from elsewhere? 

Note: I'm talking about ssh**d** here, not ssh.

**Comment by spetrosi** [comment, marked as answer]
It's at https://github.com/willshersystems/ansible-sshd
